docs/user_manual/components.md

```diff
@@ -147,6 +147,87 @@ $$
+| `c1`   | `double`  | farad (F)  | positive-sequence shunt capacitance     | &#10060; |               `> 0`                |
+
+The for the r and x matrices providing values for the neutral phase is optional. To clarify which input is values are required, please consult the tables below:
+
+| r_aa - r_cc   | r_na     | r_nb     | r_nc     | r_nn     | result   | Validation Error          |
+| ------------- | -------- | -------- | -------- | -------- | -------- | ------------------------- |
+| &#10004;      | &#10004; | &#10004; | &#10004; | &#10004; | &#10004; |                           |
+| &#10004;      | &#10004; | &#10004; | &#10004; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10004; | &#10004; | &#10004; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10004; | &#10004; | &#10060; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10004; | &#10060; | &#10060; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10060; | &#10060; | &#10060; | &#10060; | &#10004; |                           |
+| &#10060;      | &#10060; | &#10060; | &#10060; | &#10060; | &#10060; | MultiFieldValidationError |
+
+| x_aa - x_cc   | x_na     | x_nb     | x_nc     | x_nn     | result   | Validation Error          |
+| ------------- | -------- | -------- | -------- | -------- | -------- | ------------------------- |
+| &#10004;      | &#10004; | &#10004; | &#10004; | &#10004; | &#10004; |                           |
+| &#10004;      | &#10004; | &#10004; | &#10004; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10004; | &#10004; | &#10004; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10004; | &#10004; | &#10060; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10004; | &#10060; | &#10060; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10004;      | &#10060; | &#10060; | &#10060; | &#10060; | &#10004; |                           |
+| &#10060;      | &#10060; | &#10060; | &#10060; | &#10060; | &#10060; | MultiFieldValidationError |
+
+For the c-matrix values there are two options. Either provide all the required c-matrix values i.e. `c_aa` - `c_cc` or provide `c0`, `c1`. Whenver both sets are supplied the powerflow calculations will use `c0`, `c1`.
+The table below provides guidance in providing valid input.
+
+| c_aa - c_cc | c0       | c1       | result   | Validation Error          |
+| ----------- | -------- | -------- | -------- | ------------------------- |
+| &#10004;    | &#10004; | &#10004; | &#10004; |                           |
+| &#10004;    | &#10004; | &#10060; | &#10004; |                           |
+| &#10004;    | &#10060; | &#10060; | &#10004; |                           |
+| &#10060;    | &#10004; | &#10060; | &#10060; | MultiFieldValidationError |
+| &#10060;    | &#10004; | &#10004; | &#10004; |                           |
+
```

Comment:
can you please add the kron reducton here in an `Electric model` subsection like for the other parameters?